nixos/modules/tasks/network-interfaces.nix

```diff
@@ -189,6 +223,30 @@ let
         '';
       };
 
+      ipv4Routes = mkOption {
```

Comment:
I don't really like the extra `v` in `ipv[46]Routes` because the address options are just named `ip[46]`. Even without the `v`, the interface looks inconsistent to me because it is not clear that `ip[46]` is only for configuring addresses. No better idea how we should structure the `interfaceOpts` either.

Reply:
So, what about `ip{4,6}.{routes,addresses}`?

Reply:
@fpletz?